stablehlo/reference/Element.h

```diff
@@ -23,6 +23,7 @@ limitations under the License.
 #include "llvm/Support/raw_ostream.h"
 #include "mlir/IR/BuiltinTypes.h"
 #include "mlir/IR/Types.h"
+#include "stablehlo/dialect/StablehloOps.h"
```

Comment:
`Element.*` does not have to depend on `#include "stablehlo/dialect/StablehloOps.h"` for `ComparisonType` or `ComparisonDirection`. PTAL below for suggestion.